src/igvfd/schemas/changelogs/assay_term.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog for *`assay_term.json`*
 
+### Schema version 10
+* Adjust `preferred_assay_title` enum list to replace `scMito-seq` with `10x multiome with scMito-seq`.
```

Comment:
Can we add a blank line above this line?
Also, the property in assay_term is plural `preferred_assay_titles` (Measurement Set is singular).

Reply:
done